vision.md

```diff
@@ -0,0 +1,70 @@
+# Vision
+
+The goal of Typerighter is to provide a [universally available](#universal) document checking service, via  an [easy-to-use](#easy-to-use) and [minimal interface](#minimal), that is [accurate](#accurate), [transparent](#transparent), and [responsive to change](#responsive-to-change).
+
+## Universal
+
+Good proofing tools should provide value to users wherever they write.
+
+Typerighter should be available for use across our internal toolset and beyond it. It should be easy for other applications to integrate with it, regardless of platform or publishing cadence. It should just care about text.
```

Comment:
A further paragraph here being a bit more explicit about availability. Firstly by providing a non-exhaustive list of example places we'd expect to have it. Secondly a principle such as "it should be available to check any field that is ultimately published and readable by the public".